### ISSUE TITLE
Facepile: Adding a face doesn't announce itself, and arrowing right on the first item says "End of line"

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1604]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. Navigating in goes to the third persona
Expect: Should land on the first persona or object

2. Can't navigate to other personas
Expect: Arrowing should take you to other personas

3. Add face doesn't announce when a face is added
Expect: Should say "Face added, <new count>"

**Comment by betrue-final-final** [comment]
In Narrator these issues go away except for #3.

Other issues exposed:
1. Can't navigate to newly created face in "Add face" example.
2. Invoking the popup for a persona brings up silent dialog. 
3. dismissing dialog makes focus not return to the persona.

**Comment by betrue-final-final** [comment]
Issues remaining
Add face doesn't announce when a face is added. Should say "Face added"
When first entering the facepile, arrowing right at first says "End of line" and does nothing. The next arrow navigates.
